dataland-csvconverter/src/main/kotlin/org/dataland/csvconverter/csv/commonfieldparsers/EuTaxonomyCommonFieldParser.kt

```diff
@@ -49,7 +49,7 @@ class EuTaxonomyCommonFieldParser(
     /**
      * Returns the number of employees for a specific company
      */
-    fun getNumberOfEmployees(csvLineData: Map<String, String>): BigDecimal {
-        return columnMappingEuTaxonomyUtils.readCsvDecimal("numberOfEmployees", csvLineData)!!
+    fun getNumberOfEmployees(csvLineData: Map<String, String>): BigDecimal? {
```

Comment:
Is this due to some companies not having information about employees?

Reply:
The issue here was that fields were flagged as required in the data model but we do not have real data for them. Hence we reverted the change in the data model but nobody reverted the change in the csv converter. Which I now did in this PR so we are able to parse CSVs again.

Reply:
Ok